spring-batch-bigquery/pom.xml

```diff
@@ -106,39 +89,42 @@
             <version>${logback.version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>2.0.16</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <version>1.20.1</version>
             <scope>test</scope>
         </dependency>
-
     </dependencies>
 
     <build>
         <plugins>
-            <!-- Compiles source code -->
```

Comment:
Why comments that describe purpose of each plugin are removed?
I do not think that name of the plugin always describes the purpose.

Reply:
Sure, I restored them and added one for the `flatten-maven-plugin`.